drakrun/drakrun/config.py

```diff
@@ -14,7 +14,6 @@ class InstallInfo(DataClassJsonMixin):
     storage_backend: str
     disk_size: str
     iso_path: str
-    max_vms: int
```

Comment:
I haven't checked, but are we 100% sure that this won't crash if we try to deserialize an old `install.json` with `max_vms ` field?